Assets/HoloToolkit/Utilities/Scripts/ManualCameraControl.cs

```diff
@@ -28,6 +28,15 @@ public enum MouseButton
     public MouseButton MouseLookButton = MouseButton.Shift;
     public bool IsControllerLookInverted = true;
 
+    public enum ControlMode
+    {
+        Fly,
+        Walk,
+    }
+    public ControlMode controlMode = ControlMode.Fly;
+    public float controlSlowSpeed = 0.1f;
```

Comment:
These should be PascalCase if they're to be public, e.g. ControlSlowSpeed, ControlMode.